applications/ParticleMechanicsApplication/custom_strategies/strategies/MPM_strategy.h

```diff
@@ -461,61 +468,210 @@ class MPMStrategy
         mp_solving_strategy->SetEchoLevel(Level);
     }
 
-    //*********************************************************************************
-    /**OPERATIONS ACCESSIBLE FROM THE INPUT:*/
-    /*@{ */
-
-    /**
-    operation to predict the solution ... if it is not called a trivial predictor is used in which the
-    values of the solution step of interest are assumed equal to the old values
-     */
-    void Predict() override
-    {
-    }
-
     /**
-    Initialization of member variables and prior operations
+     * @brief Initialization of member variables and prior operations
```

Comment:
I suggest to only add function-explanations for overridden fcts if they are different from the base-class

otherwise if the base-class gets updated your description will be wrong